README.md

```diff
@@ -103,47 +103,19 @@ The [testing README](LIMS_IMAGE/web/tests/README.md) is found in the testing fol
 
 1. Ensure docker and docker-compose are installed, and that this repo is cloned to your machine.
 1. cd into `LIMS_IMAGE`.
-1. Run `docker-compose up --build`.
+1. For first-time setup run `docker-compose up --build db-postgres` to let the database build (make sure you wait until it says the database is ready to accept connections. It may pause after the message 'ok' but that does not mean it is finished).
```

Comment:
for first time, do you mean, that now that i've run this command the first time, I don't need to run it again?

Reply:
Yes, unless you delete the postgres data folder